src/xrEngine/EngineAPI.h

```diff
@@ -76,16 +78,17 @@ class ENGINE_API CEngineAPI
     void Destroy();
 
     void CreateRendererList();
+    bool SkipGameModuleLoading() const { return !!strstr(Core.Params, "-nogame"); }
```

Comment:
Just to be explicit =))
```suggestion
    bool CanSkipGameModuleLoading() const { return !!strstr(Core.Params, "-nogame"); }
```